Modules/Core/Common/include/itkImageSource.hxx

```diff
@@ -45,11 +45,7 @@ ImageSource<TOutputImage>::ImageSource()
   this->ProcessObject::SetNumberOfRequiredOutputs(1);
   this->ProcessObject::SetNthOutput(0, output.GetPointer());
 
-#if defined(ITKV4_COMPATIBILITY)
-  m_DynamicMultiThreading = false;
-#else
   m_DynamicMultiThreading = true;
```

Comment:
This could probably be moved to an inline initializer now.

Reply:
Fixed in different PR.